### ISSUE TITLE
[feature] support monocle time with multiple timestamps and relaxed input validation

### ISSUE THREAD

**Issue by digizeph** [bgpkit/monocle#53]
### Multiple timestamps

Should support `monocle time 1640995200 1641081600`

### Relaxed input validation

Would be great to support non-RFC3339 inputs with some reasonable default values:
* `2022-01-01` -> `2022-01-01T00:00:00Z`
* `2022-01-01T01` -> `2022-01-01T01:00:00Z`

`humantime`'s [`parse_rfc3339_weak`](https://docs.rs/humantime/latest/humantime/fn.parse_rfc3339_weak.html) could be used here.

In short, treat any string with non-numeric characters as time string and try to parse it as much as possible.
